app/main.py

```diff
@@ -1 +1,60 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    @classmethod
+    def learn_skill(cls, skill: str) -> None:
+        cls.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+
+    skills = ["JavaScript", "HTML", "CSS"]
```

Comment:
And skills should be extended, that's the purpose of inheritance
fix everywhere